packages/unmock-core/src/__tests__/corePackage.test.ts

```diff
@@ -1,21 +1,5 @@
 import { UnmockPackage } from "..";
```

Comment:
Is there a reason you all decided to use relative paths rather than absolute?

Reply:
You mean absolute like `import { UnmockPackage } from "unmock-core"`? That would work as well, but wouldn't that depend on the `moduleNameMapper` in `jest.config.js` to be configured to point to the source directory? The relative path makes it imho easier to see that one's importing from the same package.